src/test/java/se/kth/spork/LineBasedMergeTest.java

```diff
@@ -0,0 +1,27 @@
+package se.kth.spork;
```

Comment:
I'm somewhat opposed to adding new unit-ish tests unless there's strong reason for it (which I don't see, feel free to point out something I'm missing).

While this exact scenario isn't tested anywhere, line-based merge conflicts are tested in [this scenario test](https://github.com/ASSERT-KTH/spork/tree/29f187f0a85c36ad9aadf7d90cd062ca7627b791/src/test/resources/conflict/conflicting_comment_edits).

If you feel something is missing, consider either adding it to the existing test scenario I linked or creating a new scenario. Or justify why this should be unit tested.

Reply:
Sure, I can remove it… for me it's a habit more than anything else, as a way to document the behaviour of individual components, but for a tool like spork I can see it makes sense to rely on integration tests instead.